actions.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+from PyQt5 import QtWidgets
+
+import streamlink
+from constants import MUTE_CHECKBOX
+from containers import LiveStreamContainer
+
+
+def mute_all_streams(parent):
```

Comment:
Yeah, I agree with you. The thing is that we have to have the functions separate from the Application class because of the `self` binding to the UI. Couldn't get it to work by passing in `self.mute_all_streams` to the signal in `main.py`. Could be that I was doing something wrong though. 

So it was either placing the functions inside `main.py` which I already think is to hefty, or place it in a separate file. Imo it's more clearer to place it in a separate file.